File: ExBooks/app/src/main/java/ex/devs/exbooks/model/Notifications/APIService.java
package ex.devs.exbooks.model.Notifications;

import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.Headers;
import retrofit2.http.POST;

public interface APIService {

    @Headers(
            {
                    "Content-Type:application/json",
                    "Authorization:key=AAAA7wPU7Bs:APA91bEScgitbNydwjBLbcW-dGTJLp5DdQ_kMQxynQLrJKYqDOBNjYGsNk7xxGYLw-cRzmGKF_0R_cuwr7e4VJ88TivKMgB44BeQDW4mlZfZsd-JkXSRFYxDfRYw0PgmldAneW8l987U"
            }
    )

    @POST("fcm/send")
    Call<MyResponse> sendNotification(@Body Sender body);
}

File: ExBooks/app/src/main/java/ex/devs/exbooks/networking/api/ApiService.kt
package ex.devs.exbooks.networking.api

import ex.devs.exbooks.model.Notifications.MyResponse
import ex.devs.exbooks.networking.model.Sender
import ex.devs.exbooks.utils.HEADER_TOKEN
import retrofit2.Response
import retrofit2.http.Body
import retrofit2.http.Headers
import retrofit2.http.POST

interface ApiService {
    @Headers("Content-Type:application/json", "Authorization:key=$HEADER_TOKEN")
    @POST("fcm/send")
    fun sendNotification(@Body body: Sender): Response<MyResponse>
}